src/form/birth/index.ts

```diff
@@ -219,20 +220,68 @@ export const birthForm: ISerializedForm = {
           fields: [
             informantType, // Required field.
             otherInformantType(Event.Birth), // Required field.
+            getIDReaderField(
```

Comment:
@tahmidrahman-dsi I thought that we would apply these functions using functions from here: https://github.com/opencrvs/mosip/pull/26. Do we have to merge that one in first?

Reply:
Yes @euanmillar eventually we will import these from @opencrvs/mosip. please feel free to merge if the changes look okay. After the changes are published, I can import these functions here

Reply:
@tahmidrahman-dsi I spoke with Pyry, and we are going to publish multiple versions so can you close this PR and instead copy over the appropriate form changes to https://github.com/opencrvs/mosip then we can merge in and use the form functions properly in this repo.  Please note the latest WIP changes I merged in that repo. Will set up a call with you to discuss how to send the draft id, e.g. $. from the REDIRECT button.  Can you apply any updates to the functions I created with Pathum as I gather you have renamed it LINK_BUTTON now if thats correct?

Reply:
Yes that's correct. The update should be in mosip repo as well